scripts/juju-inspect/Makefile

```diff
@@ -0,0 +1,13 @@
+PROJECT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+GOOS=$(shell go env GOOS)
+GOARCH=$(shell go env GOARCH)
+
+BUILD_DIR ?= $(PROJECT_DIR)/_build
+BIN_DIR = ${BUILD_DIR}/${GOOS}_${GOARCH}/bin
```

Comment:
It'd be much easier to run if this was built to the same bin dir as other juju executables.